scripts/checksums/scrape_mysql_checksums.py

```
@@ -0,0 +1,128 @@
+    r'\((?P<file>[^)]+\.tar\.(?:gz|xz))\).*?md5">(?P<md5>[0-9a-f]{32})</code>',
+    re.DOTALL,
+)
+
+
+def fetch(url: str, cookies: str | None = None) -> str:
+    """Fetch a URL via curl and return the body."""
+    cmd = ["curl", "-sL"]
+    if cookies:
+        cmd += ["-b", cookies]
+    cmd.append(url)
+    return subprocess.check_output(cmd, text=True)
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)